mistralrs-core/src/model_loader.rs

```diff
@@ -85,10 +85,10 @@ pub fn get_model_dtype(model: &ModelSelected) -> anyhow::Result<ModelDType> {
         | ModelSelected::Lora { dtype, .. }
         | ModelSelected::XLora { dtype, .. }
         | ModelSelected::VisionPlain { dtype, .. }
-        | ModelSelected::DiffusionPlain { dtype, .. } => Ok(*dtype),
-        ModelSelected::GGUF { .. }
-        | ModelSelected::LoraGGUF { .. }
-        | ModelSelected::GGML { .. }
+        | ModelSelected::DiffusionPlain { dtype, .. }
+        | ModelSelected::GGML { dtype, .. }
+        | ModelSelected::GGUF { dtype, .. } => Ok(*dtype),
+        ModelSelected::LoraGGUF { .. }
```

Comment:
It appears that you also updated the adapter ggml/gguf models to support setting the type - but I think that is not refleced here?

Reply:
> It appears that you also updated the adapter ggml/gguf models to support setting the type - but I think that is not refleced here?

I’ve made revisions to pass the dtype to the Xlora GGUF/GGML models. However, I’m not sure how this applies to the LoRA model, as I didn’t find an explicit definition for a quantized model in LoRA. Therefore, I’ve left it in its original version.

Reply:
Sorry for the confusion - the X-LoRA and LoRA models are the same implementation, so the updates apply to "both".

Reply:
> Sorry for the confusion - the X-LoRA and LoRA models are the same implementation, so the updates apply to "both".

This have been resolved.